src/celestia/qt/qtappwin.cpp

```diff
@@ -208,6 +208,12 @@ void CelestiaAppWindow::init(const QString& qConfigFileName,
         QString dataDir = QApplication::applicationDirPath() + "/../Resources";
 #else
         QString dataDir = CONFIG_DATA_DIR;
+#ifdef APPIMAGE
+        QString appimageDir = QString::fromLocal8Bit(::getenv("APPDIR"));
```

Comment:
just a remark: in qt it's preferred to use qgetenv because on some platforms (i suppose latest windows) getenv isn't available. No need to change it here, as this code is Unix-only